protocol/protobuf/community_token_metadata.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+option go_package = "./;protobuf";
+package protobuf;
+
+message CommunityTokenMetadata {
+  enum TokenType {
+    ERC721 = 0;
+    ERC20 = 1;
+  }
```

Comment:
If we move this `message` definition to `communities.proto` then we can probably reuse a single enum for different things.

For example, here you can see a similar enum I've introduced: https://github.com/status-im/status-go/pull/3207/files#diff-b8c0db7add756912e199c0c950bd8a41a7b6c71ca47687022bf6e3ed5865be8fR40-R45